TEST: 0-add_integer.py
==========================
>>> add_integer = __import__('0-add_integer').add_integer

>>> print(add_integer(1, 2))
3
>>> print(add_integer(100, -2))
98
>>> print(add_integer(2))
100
>>> print(add_integer(100.3, -2))
98
>>> print(add_integer("J", 9))
Traceback (most recent call last):
	  ...
TypeError: a must be an integer
>>> print(add_integer(7, '9'))
Traceback (most recent call last):
	  ...
TypeError: b must be an integer
>>> print(add_integer([],[]))
Traceback (most recent call last):
	  ...
TypeError: a must be an integer
>>> print(add_integer(-2, -7))
-9
>>> print(add_integer(18.5, 5.6))
23
>>> print(add_integer())
Traceback (most recent call last):
	  ...
TypeError: add_integer() missing 1 required positional argument: 'a'
>>> print(add_integer((4,5)))
Traceback (most recent call last):
	  ...
TypeError: a must be an integer
>>> add_integer(isinstance(4, int), 3)
Traceback (most recent call last):
	  ...
TypeError: a must be an integer
>>> add_integer(True, False)
Traceback (most recent call last):
	  ...
TypeError: a must be an integer
>>> add_integer(5j, 3)
Traceback (most recent call last):
	  ...
TypeError: a must be an integer
>>> add_integer(0xFFFFFF, 3)
16777218
>>> add_integer(0b1010110, 3)
89
>>> add_integer(x, 3)
Traceback (most recent call last):
	  ...
NameError: name 'x' is not defined
